### ISSUE TITLE
Users need to know when to renew a 3 month permit

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#594]
![screen shot 2017-06-16 at 15 44 05](https://user-images.githubusercontent.com/20285115/27231594-ff736802-52aa-11e7-9e2b-3a7e1ed103e8.png)

**Comment by lizziebruce** [comment]
1.Re: vagueness of "before your 3 month permit expires" I suggest:

> 10 days before your 3 month pass expires

Gives sufficient time for 5 working days if there's a weekend in the middle of them i.e. 7 days plus 3 days leeway. There is a risk users will leave it too late anyway though and have a gap between 3 month end date and full permit start date. So best thing for user would be to let them write on the 3 month permit "Full permit applied for reference no. 1234567" in the interim period. 

2.Re: uncertainty of sufficient activity history - what are the facts for Verify being able to kick in and ID a new resident? Do we need to say something like "you'll have had to have paid a utility bill, registered to vote, registered for council tax, registered for universal credit or informed your bank of your new address"?  See also #588 redesign of the prove-identity page where we basically state that by 3 months in a new address you'll be GOV.UK Verifiable.

**Comment by EUzkuraityte** [comment]
@lizziebruce the suggested copy looks great, but I don't think we can be more specific re part 2: what the users should do in order to increase their chances in getting verified in 3 months time.. If we suggest this, it will set the expectation that by changing their utility bill address, etc. they'll get verified. In reality, the process is v complex and each piece of evidence would get scored and weighted up and in order to get verified the user would need to reach a certain score in all the categories. We also would not be able to advice them what else they need to do in addition as we don't know in which category (citizen, money or living) they come short..

**Comment by suryanaga** [comment]
Yeah, agree that we can't really be very specific on this so let's just give them a time to come back.              For now at least, this seems the best course of action -- The success of this is something to particularly monitor in private beta (which probably needs to last for at least 4 months to make sure we get these users all the through to getting their full permit).

**Comment by lizziebruce** [comment]
I've changed the application complete page to say "10 days before" instead of "shortly before":
![screen shot 2017-06-28 at 11 51 11](https://user-images.githubusercontent.com/27814324/27633695-270175fc-5bf8-11e7-9d5f-df1ff83fbbb2.png)
Changed same copy on unverified address page:
![screen shot 2017-06-28 at 11 55 00](https://user-images.githubusercontent.com/27814324/27633823-b31d0506-5bf8-11e7-9d71-e4a1f0cf8d89.png)
Also changed same copy on the new resident's permits page, the one after you enter a new address to replace the one provided by Verify:
![screen shot 2017-06-28 at 11 42 08](https://user-images.githubusercontent.com/27814324/27633557-a05b5608-5bf7-11e7-9960-b9f335ccc0ed.png)
But I can't get the end screen to show with this variable ^^, I think there's a bug in the prototype @sanjaypoyzer as it's giving 6 month/12 month option page after this. Also we should probably change the wording on the new resident permits page to say something like "As this is a new/different address for you we can only give you a 3 month permit today. 10 days before it expires, re-apply for your 6 or 12 month resident's parking permit."

**Comment by suryanaga** [comment]
@lizziebruce Come grab me when you're looking at this next and we'll work out what the bug is.
